frontend/event/create_event.html

```diff
@@ -0,0 +1,204 @@
+<div class="event-overflow-navbar">
+    <div class="create-event-mobile-top-bar">
```

Comment:
why a div with other div inside with just one component?
cant it just be only one div?